entrypoint.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash -x
 
+# Priveous build mark check
+if [ -f /versions/restorestate_failure_mark ]; then
```

Comment:
Could you please change `restorestate_failure_mark` flag to the `latest_build_failed`